src/tribol/common/Parameters.hpp

```diff
@@ -33,7 +33,7 @@ constexpr integer ANY_MESH = -1;
 /*!
  * \brief Enumerates the logging level options
  */
-enum LoggingLevel
+enum class LoggingLevel
```

Comment:
This change does not fix the compilation if DEBUG is set as a macro (the definition expands to

enum class LoggingLevel
{
UNDEFINED,
1,
...
}

Reply:
Perhaps use the same case as axom?

Reply:
I understand the issue now. I appended TRIBOL to each enum value/name and removed the class. I'm not a huge fan of this fix, but the logging needs further refactoring so this may in fact change in the future, and should address your compilation issue.